python/cuml/test/dask/test_comms.py

```diff
@@ -142,6 +142,7 @@ def test_send_recv(n_trials, ucx_cluster):
 
 
 @pytest.mark.ucx
+@pytest.mark.skip
```

Comment:
?

Reply:
@divyegala  I think this can be reverted now. I've skipped this test already in the UCX PR and provided a reason.